### PR TITLE
[reggen] Interrupt Type to CFG doc

### PR DESCRIPTION
**Pull request by eunchan**
This commit adds an interrupt type column to the interrupt section in the IP technical specification.

### CODE REVIEW

File: util/reggen/gen_cfg_html.py [hunk 1]
```diff
@@ -113,33 +112,28 @@ def gen_cfg_html(cfgs: IpBlock, outfile: TextIO) -> None:
...
-            genout(outfile,
-                   "    <tr>\n" +
-                   "      <td>" + name + "</td>\n" +
-                   "      <td>" + pkg_struct + "</td>\n" +
-                   "      <td>" + sig_type+ "</td>\n" +
-                   "      <td>" + act + "</td>\n" +
-                   "      <td>" + width + "</td>\n" +
-                   "      <td>" + desc + "</td>\n" +
-                   "    </tr>\n")
+            genout(
```
Comment (timothytrippel) [inline util/reggen/gen_cfg_html.py:115]: nit: was this hit by an autoformatter? might be better to turn off formatting for these lines (and others in this file) as I think they were more readable before

Comment (timothytrippel) [reply util/reggen/gen_cfg_html.py:115]: ah just saw the second commit message.

Comment (eunchan) [reply util/reggen/gen_cfg_html.py:115]: Hum, `flake8` and `yapf` do not work well together.. I will skip yapf part.. Are `yapf` and `isort` optional now?

Comment (timothytrippel) [reply util/reggen/gen_cfg_html.py]: I don't think they are enforced in CI at all, so no hehe.

**Pull request merged**